### PR TITLE
Fixup documentation for cpvpa.

### PR DESCRIPTION
**Pull request by dnardo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+[![Go Report Card](https://goreportcard.com/badge/github.com/kubernetes-incubator/cluster-proportional-vertical-autoscaler)](https://goreportcard.com/report/github.com/kubernetes-incubator/cluster-proportional-vertical-autoscaler)
 
 ## Overview
 
-This container image watches over the number of schedulable nodes and cores of the cluster and resizes
-the number of replicas for the required resource. This functionality may be desirable for applications
-that need to be autoscaled with the size of the cluster, such as DNS and other services that scale
-with the number of nodes/pods in the cluster.
+This container image watches over the number of nodes and cores of the cluster and resizes
+the resource limits and requests for the a DaemonSet, ReplicaSet, or Deployment. This functionality
```
Comment (thockin) [inline README.md]: "for the a"

File: README.md [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
 ## Overview
 
-This container image watches over the number of schedulable nodes and cores of the cluster and resizes
-the number of replicas for the required resource. This functionality may be desirable for applications
-that need to be autoscaled with the size of the cluster, such as DNS and other services that scale
-with the number of nodes/pods in the cluster.
+This container image watches over the number of nodes and cores of the cluster and resizes
+the resource limits and requests for the a DaemonSet, ReplicaSet, or Deployment. This functionality 
+may be desirable for applications where resources need to be autoscaled with the size of the cluster, 
+and scaling horizontally (number of replicas) .
```
Comment (thockin) [inline README.md]: This doesn't apply (yet) ?

File: README.md [hunk 3]
```diff
@@ -39,105 +39,138 @@ The code in this module is a Kubernetes Golang API client that, using the defaul
...
 ```
-data:
-  linear: |-
-    {
-      "coresPerReplica": 2,
-      "nodesPerReplica": 1,
-      "min": 1,
-      "max": 100,
-      "preventSinglePointFailure": true
+someTarget: {
```
Comment (thockin) [inline README.md]: needs to be quoted

Comment (thockin) [inline README.md]: this is the container name within the pod, not just someTarget

Comment (thockin) [inline README.md]: might be a good example to have 2 containers in a pod

File: README.md [hunk 4]
```diff
@@ -39,105 +39,138 @@ The code in this module is a Kubernetes Golang API client that, using the defaul
...
-replicas = min(replicas, max)
-replicas = max(replicas, min)
+## Running the cluster-proportional-vertical-autoscaler
+This repo includes an example yaml files in the "Example" folder that can be used as examples demonstrating 
+how to use the vertical autoscaler.
+
+For example, consider a Deployment that needs to scale its resources (cpu, memory, etc...) proportional to the number of
+cores in a cluster.
+
+```apiVersion: extensions/v1beta1
```
Comment (thockin) [inline README.md]: newline after ```

File: README.md [hunk 5]
```diff
@@ -39,105 +39,138 @@ The code in this module is a Kubernetes Golang API client that, using the defaul
...
+kind: Deployment
+metadata:
+  name: thing
+  namespace: kube-system
+  labels:
+    kubernetes.io/cluster-service: "true"
```
Comment (thockin) [inline README.md]: minimize - strike anything not needed to demontsrate

File: README.md [hunk 6]
```diff
@@ -39,105 +39,138 @@ The code in this module is a Kubernetes Golang API client that, using the defaul
...
+        k8s-app: thing
+    spec:
+      containers:
+      - image: thing/thing-v0.0.1
+        name: thing
+        ports:
+        - containerPort: 4000
+          name: thing
+          protocol: TCP
+        resources:
```
Comment (thockin) [inline README.md]: don't put resources here, better to let the scaler own it.

File: README.md (hunk 1 above)
Comment (dnardo) [inline README.md]: done.

File: README.md (hunk 2 above)
Comment (dnardo) [inline README.md]: yes forgot to remove that, updated.

File: README.md (hunk 3 above)
Comment (dnardo) [inline README.md]: done.

Comment (dnardo) [inline README.md]: PTAL

File: README.md (hunk 4 above)
Comment (dnardo) [inline README.md]: done

File: README.md (hunk 6 above)
Comment (dnardo) [inline README.md]: PTAL, I think that is the minimal, let me know if more can should be pulled out.

**Comment by dnardo** [comment]
Addressed comments, PTAL

File: README.md [hunk 7]
```diff
@@ -39,105 +39,135 @@ The code in this module is a Kubernetes Golang API client that, using the defaul
...
+  template:
+    metadata:
+      labels:
+        k8s-app: thing-autoscaler
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      containers:
+      - name: autoscaler
+        image: gcr.io/google_containers/cluster-proportional-vertical-0autoscaler-amd64:1.0.0
```
Comment (thockin) [inline README.md]: errant 0 in "0autoscaler"

File: README.md [hunk 8]
```diff
@@ -39,105 +39,135 @@ The code in this module is a Kubernetes Golang API client that, using the defaul
...
+    },
+  }
+}
 ```
 
-The ladder controller gives out the desired replicas count by using a step function.
-The step ladder function uses the datapoint for core and node scaling from the ConfigMap.
-The lookup which yields the higher number of replicas will be used as the target scaling number.
+## Running the cluster-proportional-vertical-autoscaler
+This repo includes an example yaml files in the "Example" folder that can be used as examples demonstrating
```
Comment (thockin) [inline README.md]: example or Example.  dirs should be lower-case, I guess.

**Pull request merged**